src/routes/rules/data/rulesData.js

```diff
@@ -37,102 +32,90 @@ const rules = [
 const royals = [
   {
     title: 'rules.royals.king',
-    icon: 'crown',
     description: 'rules.royals.kingDescription',
     staticImg: '/img/rulesView/royals_king.svg',
     animatedImg: 'https://github.com/cuttle-cards/cuttle-assets/blob/main/assets/king.gif?raw=true',
   },
   {
     title: 'rules.royals.queen',
-    icon: 'crown',
     description: 'rules.royals.queenDescription',
     staticImg: '/img/rulesView/royals_queen.svg',
     animatedImg: 'https://github.com/cuttle-cards/cuttle-assets/blob/main/assets/queen.gif?raw=true',
   },
   {
     title: 'rules.royals.jack',
-    icon: 'crown',
     description: 'rules.royals.jackDescription',
     staticImg: '/img/rulesView/royals_jack.svg',
     animatedImg: 'https://github.com/cuttle-cards/cuttle-assets/blob/main/assets/jack.gif?raw=true',
   },
+  {
+    title: 'rules.royals.eight',
+    description: 'rules.royals.eightDescription',
+    staticImg: '/img/rulesView/oneoffs_eight.svg',
+    animatedImg: 'https://github.com/cuttle-cards/cuttle-assets/blob/main/assets/glasses.gif?raw=true',
+  },
```

Comment:
I moved glasses eights into the 'Royals' section and updated the translation files to reflect this as well, in ordering and content